plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -224,7 +224,19 @@ private function get_feature_definitions() {
 					'is_legacy'          => true,
 					'is_experimental'    => false,
 				),
-				'hpos_fts_indexes'     => array(
+				'site_visibility_badge' => array(
+					'name'               => __( 'Site visibility badge', 'woocommerce' ),
+					'description'        => __(
+						'Enable the site visibility badge in the WordPress admin bar',
+						'woocommerce'
+					),
+					'enabled_by_default' => true,
+					'disable_ui'         => false,
+					'is_legacy'          => true,
```

Comment:
This isn't a legacy feature but `is_legacy` seems necessary to allow users to toggle the feature in the UI. Otherwise it disables the checkbox. It seems like other new features are following the same approach.